base/array.jl

```diff
@@ -2404,7 +2408,8 @@ function findall(pred::Fix2{typeof(in),<:Union{Array{<:Real},Real}}, x::Array{<:
 end
 # issorted fails for some element types so the method above has to be restricted
 # to element with isless/< defined.
-findall(pred::Fix2{typeof(in)}, x::Union{AbstractArray, Tuple}) = _findin(x, pred.x)
+findall(pred::Fix2{typeof(in)}, x::AbstractArray) = _findin(x, pred.x)
```

Comment:
unrelated change?

Reply:
I think it's required to avoid an ambiguity, since `::Fix2` is more specific than `::Function`, but `::AbstractArray` is more specific than `::Union{AbstractArray, Tuple}`.